README.md

```diff
@@ -132,9 +132,10 @@ as the return type of a controller method, where you can add multiple templates.
 @GetMapping("/partials/main-and-partial")
 public HtmxResponse getMainAndPartial(Model model){
     model.addAttribute("userCount", 5);
-    return new HtmxResponse()
-        .addTemplate("users-list")
-        .addTemplate("users-count");
+    return HtmxResponse.builder()
+        .template("users-list")
```

Comment:
These examples should also use the `view()` method.

Reply:
Done.